### PR TITLE
Explain meta-label process with analyst and tactician

### PR DESCRIPTION
**Pull request by remyroche**
Adds a rolling, triggerless model for continuous market path and direction prediction.

This enables proactive trading decisions (entry, sizing, exit) by predicting trend emergence, continuation, and reversal, along with price direction, without relying on meta-label event triggers. The previous model only classified post-event paths around a trigger.

---
<a href="https://cursor.com/background-agent?bcId=bc-68829801-9e36-418a-b39e-9b9de24d595d">
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://cursor.com/open-in-cursor-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://cursor.com/open-in-cursor-light.svg">
    Open in Cursor
  </picture>
</a>
<a href="https://cursor.com/agents?id=bc-68829801-9e36-418a-b39e-9b9de24d595d">
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://cursor.com/open-in-web-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://cursor.com/open-in-web-light.svg">
    Open in Web
  </picture>
</a>

### CODE REVIEW

File: src/transition/multitask_rf.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        for name, model in self.models.items():
+            try:
+                if hasattr(model, "predict_proba"):
+                    proba = model.predict_proba(X)
+                    classes = getattr(model, "classes_", [])
+                    out[name] = {str(c): proba[:, i].tolist() for i, c in enumerate(classes)}
+                else:
+                    out[name] = model.predict(X).tolist()
+            except Exception:
+                out[name] = []
```
[the remark this answers is not in the document]
Comment (remyroche) [reply src/transition/multitask_rf.py:356]: @cursoragent fix

File: src/transition/rolling_window_dataset.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                "path_class": pc,
+                "X_pre_states": X_states,
+                "X_pre_numeric": np.zeros((pre, 0), dtype=float),
+                "Y_post_states": Y_states,
+                "Y_post_returns": y_rets.copy(),
+                "rf_features": rf_feats,
+                **dir_targets,
+            })
+            if self.rw_cfg.max_samples and len(samples) >= self.rw_cfg.max_samples:
+                break
```
[the remark this answers is not in the document]
Comment (remyroche) [reply src/transition/rolling_window_dataset.py:144]: @cursoragent fix

File: src/transition/multitask_rf.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        if not self.cfg.enabled or not samples:
+            return {"trained": False}
+        X = self._assemble_X(samples)
+
+        results: Dict[str, Any] = {"trained": True}
+
+        # 1) Path class
+        y_pc = pd.Series([str(s.get("path_class", "end_of_trend")) for s in samples])
+        X_pc, y_pc = self._cap(X, y_pc)
+        Xtr, Xva, ytr, yva = train_test_split(X_pc, y_pc, test_size=0.2, random_state=self.cfg.random_state, stratify=y_pc)
```
[the remark this answers is not in the document]
Comment (remyroche) [reply src/transition/multitask_rf.py]: @cursoragent fix
